src/components/breadcrumbs/breadcrumbs.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { Breadcrumbs } from "./breadcrumbs";
+import { nodesTrail } from "./breadcrumbs.mock";
+
+describe("Parent links", () => {
+  const breadcrumbAssumptions = [
+    { title: "Home", href: "/" },
+    { title: "News", href: "/news/" },
+    { title: "Podcasts", href: "/news/podcasts/" },
+    {
+      title: "Why the world needs more women leaders in science and business",
+      href: "/news/podcasts/women-leaders/",
+    },
+  ];
+
+  it("it should link to parent pages", () => {
```

Comment:
Can we remove the "it" at the start of the it function? 

This may just be personal opinion, but I feel tests inside a describe are better to read and understand when the "describe" text plus the "it" text make a valid title for the test (e.g. "Parent links should link to parent pages" rather than "Parent links it should link to parent pages")

Reply:
Ah for sure, have pushed an update for this now.